### DISCUSSION TITLE
Slack update For 2022?

### DISCUSSION THREAD

**Discussion by Jamie-Lee-Campbell** [linuxdeepin/developer-center#3379] [Q&A | 问答板块]
Has anyone been able to get Slack working? The app store version is outdated and I tried installing the .deb from [slack.com](https://slack.com/downloads/linux) using package installer and using dpkg on the CLI but no luck.

**Comment by BLumia** [comment]
Could you provide the log when you try to install the deb package from CLI? thanks!

cc @Zeno-sole

**Reply by Jamie-Lee-Campbell** [reply]
sudo dpkg -i /home/King/Downloads/slack-desktop-4.27.156-amd64.deb
(Reading database ... 239224 files and directories currently installed.)
Preparing to unpack .../slack-desktop-4.27.156-amd64.deb ...
Unpacking slack-desktop (4.27.156) ...
dpkg: error processing archive /home/king/Downloads/slack-desktop-4.27.156-amd64.deb (--install):
 trying to overwrite '/usr/share/lintian/overrides/slack', which is also in package slack 1:0.15.2-9
Processing triggers for bamfdaemon (0.5.4.1-1+eagle) ...
Rebuilding /usr/share/applications/bamf-2.index...
Processing triggers for desktop-file-utils (0.23-4) ...
Processing triggers for mime-support (3.62) ...
Errors were encountered while processing:
 /home/King/Downloads/slack-desktop-4.27.156-amd64.deb

======= Using apt 

sudo apt install /home/King/Downloads/slack-desktop-4.27.156-amd64.deb
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'slack-desktop' instead of '/home/King/Downloads/slack-desktop-4.27.156-amd64.deb'
Suggested packages:
  gir1.2-gnomekeyring-1.0 libgnome-keyring0
The following NEW packages will be installed:
  slack-desktop
0 upgraded, 1 newly installed, 0 to remove and 8 not upgraded.
Need to get 0 B/66.9 MB of archives.
After this operation, 148 MB of additional disk space will be used.
Get:1 /home/King/Downloads/slack-desktop-4.27.156-amd64.deb slack-desktop amd64 4.27.156 [66.9 MB]
N: Ignoring file 'sury-php.listPassword' in directory '/etc/apt/sources.list.d/' as it has an invalid filename extension
(Reading database ... 239224 files and directories currently installed.)
Preparing to unpack .../slack-desktop-4.27.156-amd64.deb ...
Unpacking slack-desktop (4.27.156) ...
dpkg: error processing archive /home/King/Downloads/slack-desktop-4.27.156-amd64.deb (--unpack):
 trying to overwrite '/usr/share/lintian/overrides/slack', which is also in package slack 1:0.15.2-9
Errors were encountered while processing:
 /home/King/Downloads/slack-desktop-4.27.156-amd64.deb
N: Ignoring file 'sury-php.listPassword' in directory '/etc/apt/sources.list.d/' as it has an invalid filename extension
N: Download is performed unsandboxed as root as file '/home/King/Downloads/slack-desktop-4.27.156-amd64.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Reply by Jamie-Lee-Campbell** [reply]
If you can provide a fix I would greatly appreciate it. 谢谢

**Reply by Zeno-sole** [reply]
@tsic404  Please update the app

**Reply by tsic404** [reply]
> dpkg: error processing archive /home/King/Downloads/slack-desktop-4.27.156-amd64.deb (--unpack):
trying to overwrite '/usr/share/lintian/overrides/slack', which is also in package slack 1:0.15.2-9

you seem to install slack-desktop and slack at same time, try to install slack-desktop after remove slack
